presto-main/src/test/java/io/prestosql/sql/query/TestColumnMask.java

```diff
@@ -425,4 +425,32 @@ public void testShowStats()
                         "(cast('clerk' AS varchar), cast(15000 AS double), cast(1000 AS double), cast(0 AS double), cast(null AS double), cast(null AS varchar), cast(null AS varchar))," +
                         "(null, null, null, null, 15000, null, null)");
     }
+
+    @Test
```

Comment:
do we have a test where we mask more than one columns?

Reply:
Added